controllers/cloud.redhat.com/clowdenvironment_controller.go

```diff
@@ -166,9 +166,9 @@ func (r *ClowdEnvironmentReconciler) Reconcile(req ctrl.Request) (ctrl.Result, e
 	}
 	SetDeploymentStatus(ctx, &proxyClient, &env)
 
-	env.Status.Ready = false
-	if env.Status.Deployments.ManagedDeployments == env.Status.Deployments.ReadyDeployments {
-		env.Status.Ready = true
+	notReady := env.IsReady() == false
+	if notReady {
+		r.Log.Info("ClowdEnvironment is not ready")
```

Comment:
We still need to actually set the status - else the Ready boolean will never flip to true